atom/browser/native_window_views.cc

```diff
@@ -490,11 +496,15 @@ bool NativeWindowViews::IsMaximizable() {
 #endif
 }
 
-void NativeWindowViews::SetFullScreenable(bool maximizable) {
+void NativeWindowViews::SetFullScreenable(bool fullscreenable) {
+  #if defined(OS_WIN)
+    FlipWindowStyle(GetAcceleratedWidget(), false, WS_MAXIMIZEBOX);
```

Comment:
According to the `BrowserWindow` [docs](https://github.com/atom/electron/blob/master/docs/api/browser-window.md#new-browserwindowoptions), `fullscreen: false` should disable the maximize button on Windows, that's why I included this. Shall I remove it and correct the docs?